src/main/java/graphql/annotations/processor/retrievers/fieldBuilders/field/FieldDataFetcherBuilder.java

```diff
@@ -33,23 +35,25 @@
 
 import static graphql.Scalars.GraphQLBoolean;
 import static graphql.annotations.processor.util.ConnectionUtil.getConnectionDataFetcher;
+import static graphql.annotations.processor.util.ConnectionUtil.getSimpleConnectionDataFetcher;
 import static java.util.Objects.nonNull;
 
+@SuppressWarnings("ALL")
 public class FieldDataFetcherBuilder implements Builder<DataFetcher> {
     private Field field;
     private DataFetcherConstructor dataFetcherConstructor;
     private GraphQLType outputType;
     private TypeFunction typeFunction;
     private ProcessingElementsContainer container;
-    private boolean isConnection;
+    private TypesConnectionChecker typesConnectionChecker;
```

Comment:
why not using a "connectionType" field or something like that? and it will be null if no connection is sent..
Because the TypesConnectionChecker is a little bit confusing